jordan_bundy/server.js

```diff
@@ -0,0 +1,24 @@
+var http = require('http');
+var app = require('express')();
+var fs = require('fs');
+
+app.get('/data', function(req, res) {
+  var readStream = fs.createReadStream(__dirname + '/data/input.json');
+  res.writeHead(200, {'content-type':'application/json'});
+  readStream.pipe(res);
+});
+
+app.post('/data', function(req, res) {
+  var writeStream = fs.createWriteStream(__dirname + '/data/input.json');
+
```

Comment:
So, this code will allow me to save a single post to a file called `input.json` - but the goal is to be able to make multiple posts, named whatever I want (file names based on the url I post to).  For example, if I post to `/data/molly_note`, it should make `molly_note.json` without overwriting any other files.  Then I can make a get request to `/data/molly_note` and it should spit back the file contents.

You are close!  Just add the filename capability and you should get full credit.